synapse/federation/transport/server/_base.py

```diff
@@ -309,7 +309,7 @@ async def new_func(
                 raise
 
             # update the active opentracing span with the authenticated entity
-            set_tag("authenticated_entity", origin)
+            set_tag("authenticated_entity", str(origin))
```

Comment:
Jaeger [automatically casts things to string](https://github.com/jaegertracing/jaeger-client-python/blob/4ee7dcc1285c1b40916df20aeb5a92e112177174/jaeger_client/thrift.py#L56-L60), but according to the `opentracing` API these must be simple types.